### ISSUE TITLE
Testnet stats telling lies about genesis

### ISSUE THREAD

**Issue by telezhnaya** [near/near-explorer#746]
https://explorer.testnet.near.org/stats

It says that genesis started July 31, 2020
But in fact, in the Indexer DB, we have data from April 1, 2021

**Comment by shelegdmitriy** [comment]
We query for genesis time start from [rpc](https://github.com/near/near-explorer/blob/master/backend/src/near.js#L26-L29):
```
const networkProtocolConfig = await nearRpc.sendJsonRpc(
  "EXPERIMENTAL_protocol_config",
  { finality: "final" }
);
``` 

If for mainnet rpc returns correct genesis time why this happens on testnet? Also how can we get the genesis time from indexer DB? I can't see such field in [indexer-for-explorer](https://github.com/near/near-indexer-for-explorer#database-structure) structure.

**Comment by telezhnaya** [comment]
@shelegdmitriy I simply run 
```
select to_timestamp(div(block_timestamp, 1000 * 1000 * 1000))::date 
from blocks 
order by blocks.block_timestamp 
limit 1
```

**Comment by telezhnaya** [comment]
@bowenwang1996 could you please check why RPC gives `July 31 2020` for Testnet genesis?

**Comment by frol** [comment]
@bowenwang1996 Well, it is not RPC, but the genesis that states that it dates from `July 31 2020` while we did the hard-fork this spring. I feel like we can ignore genesis time and instead say that the first produced block was at ... What do you think?

**Comment by bowenwang1996** [comment]
> @bowenwang1996 Well, it is not RPC, but the genesis that states that it dates from July 31 2020 while we did the hard-fork this spring. I feel like we can ignore genesis time and instead say that the first produced block was at ... What do you think?

Yeah we did not update genesis time probably unfortunately :( I think as Frol said, we could use the timestamp of the first block instead

**Comment by shelegdmitriy** [comment]
So from where can I query for the timestamp of the first block? @frol
